pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -32,10 +32,19 @@ func (n *NetworkSpec) IsMultus() bool {
 	return n.Provider == NetworkProviderMultus
 }
 
-// IsHost get whether to use host network provider. This method also preserve
-// compatibility with the old HostNetwork field.
+// IsHost can be used to determine if pods should use host networking by
+// checking whether the cephCluster is configured to use the "host" network provider.
+// This method also maintains compatibility with the old hostNetwork setting
 func (n *NetworkSpec) IsHost() bool {
-	return (n.HostNetwork && n.Provider == NetworkProviderDefault) || n.Provider == NetworkProviderHost
+	return (n.HostNetwork && n.Provider == NetworkProviderDefault) || n.Provider == NetworkProviderHost || n.Provider == NetworkProviderHostStrict
+}
+
+// IsHostStrict determines whether the cephCluster is configured to use the use host
```

Comment:
CephCluster is a resource `Kind` that is usually denoted with upper camel case for user-facing docs. Godocs in `pkg/apis` are user-facing because they usually get rendered into CRDs.

```suggestion
// IsHostStrict determines whether the CephCluster is configured to use the use host
```